addons/block-cherry-picking/userscript.js

```diff
@@ -0,0 +1,41 @@
+export default async function ({ addon, global, console }) {
+  const BlocklyInstance = await addon.tab.traps.getBlockly();
+  const originalObject = BlocklyInstance.Block.prototype.unplug;
+  let invertDrag = addon.settings.get("invertDrag");
+
+  // Necessary to detect the CTRL/CMD key
+  let ctrlKeyPressed = false;
+  document.addEventListener("keydown", function (e) {
+    if (e.ctrlKey) {
+      ctrlKeyPressed = true;
+    }
+  });
+  document.addEventListener("keyup", function (e) {
+    if (!e.ctrlKey) {
+      ctrlKeyPressed = false;
+    }
+  });
+  document.addEventListener("keydown", function (e) {
+    if (e.metaKey) {
+      ctrlKeyPressed = true;
+    }
+  });
+  document.addEventListener("keyup", function (e) {
+    if (!e.metaKey) {
+      ctrlKeyPressed = false;
+    }
+  });
```

Comment:
any reason these can't be shortened to eg.?

```js
  document.addEventListener("keydown", function (e) {
    ctrlKeyPressed = e.ctrlKey || e.metaKey;
  });
```

Reply:
I just copied it over from the Ctrl + click to run scripts addon -- I'll definitely condense that to what you're suggesting.